src/components/grid-layout/cards/diagrams/networkAreaDiagram/network-area-diagram-content.tsx

```diff
@@ -252,6 +252,7 @@ function NetworkAreaDiagramContent(props: NetworkAreaDiagramContentProps) {
                     equipmentSubtype
                 );
             };
+            setShouldDisplayTooltip(false);
```

Comment:
?

Reply:
This is needed to hide the tooltip when right clicking on an element and choosing to display the element in the table.